solr/core/src/java/org/apache/solr/util/ExportTool.java

```diff
@@ -216,32 +216,32 @@ void end() throws IOException {
       Option.builder("url")
           .hasArg()
           .required()
-          .desc("Address of the collection, example http://localhost:8983/solr/gettingstarted")
+          .desc("Address of the collection, example http://localhost:8983/solr/gettingstarted.")
```

Comment:
I think this will be confusing to people if the period is part of the URL or not (if they are copy pasting from a tutorial perhaps.

Reply:
what would you say to wrapping it in single quotes?  Some other places that appears to be the pattern.   

Address of the collection, example 'http://localhost:8983/solr/gettingstarted'.

Or maybe, I'm just being pedantic about my periods and should skip it!

Reply:
also, we do this a lot for the ZooKeeper host:  

```
```

Reply:
Hey @madrob what do you think of my comments?  I can go either way, or not change it at all!  Love to get these changes in soon.